### PR TITLE
[netatmo] Webhook events are dispatched but group id is missing

### PR DESCRIPTION
**Pull request by clinique**
Resolves issue #17660 

Being the first opening this issue, I guess this feature is not that much used :-)

### CODE REVIEW

File: bundles/org.openhab.binding.netatmo/README.md [hunk 1]
```diff
@@ -649,18 +649,19 @@ Person things are automatically created in discovery process for all known perso
...
-| person        | last-seen    | DateTime  | Moment when this person was last seen                  |
-| last-event    | subtype      | String    | Sub-type of event                                      |
-| last-event    | message      | String    | Last event message from this person                    |
-| last-event    | time         | DateTime  | Moment of the last event for this person               |
-| last-event    | snapshot     | Image     | Picture of the last event for this person              |
-| last-event    | snapshot-url | String    | URL for the picture of the last event for this person  |
-| last-event    | camera-id    | String    | ID of the camera that triggered the event              |
+| Channel Group  | Channel ID   | Item Type | Description                                                              |
+| -------------- | ------------ | --------- | ------------------------------------------------------------------------ |
+| security-event | home-event   |           | Trigger channel which is triggered an event for this person is triggered |
```
Comment (lsiepel) [inline bundles/org.openhab.binding.netatmo/README.md:654]: _Trigger channel which is triggered an event for this person is triggered_

That might need some improvement.

Comment (clinique) [reply bundles/org.openhab.binding.netatmo/README.md:654]: An event for this person has been triggered ?

Comment (lsiepel) [reply bundles/org.openhab.binding.netatmo/README.md:654]: On what occasions? When the person enters or leaves?

Comment (clinique) [reply bundles/org.openhab.binding.netatmo/README.md:654]: When the person has been identified by a camera.

Comment (lsiepel) [reply bundles/org.openhab.binding.netatmo/README.md:654]: Suggestion:
_Trigger channel which is triggered by an event for this person (PERSON, PERSON_AWAY, PERSON_HOME)_

**Review by lsiepel — APPROVED** [review]
One textual comment is open, otherwise LGTM.
Would be nice if @lolodomo can confim.

**Review by lsiepel — APPROVED** [review]

**Pull request merged**